examples/migrate_server_profile_from_bay1_to_bay12.yml

```diff
@@ -0,0 +1,114 @@
+###
+# Copyright 2017 Hewlett Packard Enterprise Development LP
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# You may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+###
+- hosts: all
+  vars:
+    config: '{{ playbook_dir }}/oneview_config.json'
+    server_hardware_type_name: "SY 480 Gen9 3"
+    enclosure_group_name: "SYN107_EG"
+    server_hardware_name1: "2-USE722C99W-M, bay 1"
+    server_hardware_name2: "2-USE722C99W-M, bay 11"
+    profile_name1: "profile_san"
+    management_network: "Management"
+    SAN_A_network: "SAN-A"
+    SAN_B_network: "SAN-B"
+
+  tasks:
+
+    - name: Power Off the server hardware
```

Comment:
Which server hardware? You specified two server hardwares.

Also, don't you need to power off both server hardwares?

Reply:
Also, with the current module, I guess you don't need to care if the server hardwares are powered off while operating a server profile, the code already checks and power them off if necessary.

Reply:
^ This. 
Since it should be working independently of the power state of the server hardwares, this example is illustrating something we don't really want to be doing.

Reply:
Ok, I removed the first task and updated the description.